src/CachePlugin.php

```diff
@@ -276,9 +333,11 @@ private function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'cache_lifetime' => 86400 * 30, // 30 days
             'default_ttl' => 0,
+            //Deprecated as of v1.3, to be removed in v2.0. Use respect_response_cache_directives instead
             'respect_cache_headers' => true,
             'hash_algo' => 'sha1',
             'methods' => ['GET', 'HEAD'],
+            'respect_response_cache_directives' => ['no-cache', 'private', 'max-age', 'no-store'],
```

Comment:
looks like adding no-cache is a change over current behaviour. i think we can consider that a bugfix. but please add a note to the changelog so people have a chance of seeing the change.